src/briefcase/ios.py

```diff
@@ -165,7 +165,8 @@ def set_device_target(self):
                             self.device_name = self.device['name']
                         except:
```

Comment:
At column 25: (E722) do not use bare except'

Reply:
I believe I do not have the prerequisite knowledge to use it